cfpb/consumerfinance.gov
# Upgrade to node 18

Migrating to node 18 will allow use to update to latest Jest.

## Changes

- Upgrade to node 18 from 16


## How to test this PR

1. PR checks should pass.
2. Update your local node via `nvm install 18 && nvm use 18`
